SDL1: test_freetype_Font_bgcolor failing

```
======================================================================
FAIL: test_freetype_Font_bgcolor (__main__.FreeTypeFontTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Program Files\Python38\Lib\site-packages\pygame-2.0.0.dev9-py3.8-win-amd64.egg\pygame\tests\freetype_test.py", line 1247, in test_freetype_Font_bgcolor
    self.assertEqual(s.get_at((0, 0)), transparent_black)
AssertionError: (255, 255, 255, 2) != (0, 0, 0, 0)

----------------------------------------------------------------------
```

System details:
- os: windows 10 (64bit)
- python: 3.8.1 (64bit)
- pygame: 2.0.0.dev9 (SDL: 1.2.15) at 0e761fd0541c53c5a20c6756b6c26d9626ef5c06

--- a/test/freetype_test.py
+++ b/test/freetype_test.py
@@ -1230,7 +1230,7 @@ class FreeTypeFontTest(unittest.TestCase):
         self.assertRaises(AttributeError, setattr, f, "fgcolor", None)
 
     def test_freetype_Font_bgcolor(self):
-        f = ft.Font(None, 12)
+        f = ft.Font(None, 32)
         zero = "0"  # the default font 0 glyph does not have a pixel at (0, 0)
         f.origin = False
         f.pad = False

--- a/test/freetype_test.py
+++ b/test/freetype_test.py
@@ -679,7 +679,7 @@ class FreeTypeFontTest(unittest.TestCase):
         rrect = font.render_to(surf, rect, "FoobarBax", color, None, size=24)
         self.assertEqual(rrect.top, rrect.height)
         self.assertNotEqual(rrect.size, rect.size)
-        rrect = font.render_to(surf, (int(20.1), int(18.9)), "FoobarBax", color, None, size=24)
+        rrect = font.render_to(surf, (20.1, 18.9), "FoobarBax", color, None, size=24)
         ## self.assertEqual(tuple(rend[1].topleft), (20, 18))
 
         rrect = font.render_to(surf, rect, "", color, None, size=24)

--- a/test/freetype_test.py
+++ b/test/freetype_test.py
@@ -679,7 +679,7 @@ class FreeTypeFontTest(unittest.TestCase):
         rrect = font.render_to(surf, rect, "FoobarBax", color, None, size=24)
         self.assertEqual(rrect.top, rrect.height)
         self.assertNotEqual(rrect.size, rect.size)
-        rrect = font.render_to(surf, (20.1, 18.9), "FoobarBax", color, None, size=24)
+        rrect = font.render_to(surf, (int(20.1), int(18.9)), "FoobarBax", color, None, size=24)
         ## self.assertEqual(tuple(rend[1].topleft), (20, 18))
 
         rrect = font.render_to(surf, rect, "", color, None, size=24)

